src/TheFox/Smtp/Client.php

```diff
@@ -101,9 +105,9 @@ public function getPort(){
 	
 	public function setIpPort($ip = '', $port = 0){
 		// @codeCoverageIgnoreStart
-		if(!TEST){
```

Comment:
Why has this been deactivated? The `getPeerName()` is used to get the `$ip` and `$port`.